projects/habitat_ovmm/README.md

```diff
@@ -9,11 +9,16 @@
 
 ## Dataset Setup
 
-Run `git lfs install` to install Git LFS, which is used to manage the OVMM dataset.
+Run `git lfs install` to install Git LFS, which is used to manage the OVMM dataset. Please sign in [here](https://huggingface.co/datasets/hssd/hssd-hab/tree/ovmm) and accept the license for using HSSD scenes.
+
+You can then use the following commmand to install download data:
+```
+$HOME_ROBOT_ROOT/projects/habitat_ovmm/install.sh
+```
 
 ### Scene dataset setup 
 
```

Comment:
Do we need the instructions in README as well, now that we have moved everything under a script?

Reply:
I was thinking if something went wrong with the script this might be useful? Maybe we should rewrite things and just add more messages to the script?

Reply:
Sure, we can keep it. We just need to make it clear that only one of these needs to be done: either run the script or follow the instructions

Reply:
Right now it still looks like someone has to do the manual steps, maybe clarify that they don't need to? Also move the section about signing the license to before the install script.